pallets/band/Cargo.toml

```diff
@@ -21,6 +21,7 @@ frame-benchmarking = { git = "https://github.com/sora-xor/substrate.git", branch
 sp-std = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
 common = { path = "../../common", default-features = false }
 hex-literal = "0.3.1"
+pallet-timestamp = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
```

Comment:
Do you really need it?
You uses UnixTime trait and binds it with pallet-timestamp only in Runtime. If you need it only for `mock`, then move it to `dev-dependencies`